### ISSUE TITLE
Error: Request failed with status code 500

### ISSUE THREAD

**Issue by denkristoffer** [siddharthkp/bundlesize#41]
Hi, we're trying to set up bundlesize with our CI but on every build we're seeing this `Error: Request failed with status code 500`.

This is what it's complaining about:

```
data: '<!DOCTYPE html>\n<html lang="en">\n<head>\n<meta charset="utf-8">\n<title>Error</title>\n</head>\n<body>\n<pre>Error: Firebase.child failed: First argument was an invalid path: &quot;5b6276128b031ac95462607a8060cfe85b60ca1c/Onlaw/onlaw.dk&quot;. Paths must be non-empty strings and can&#39;t contain &quot;.&quot;, &quot;#&quot;, &quot;$&quot;, &quot;[&quot;, or &quot;]&quot;<br> &nbsp; &nbsp;at Sd (/home/nowuser/src/node_modules/firebase/database-node.js:106:59)<br> &nbsp; &nbsp;at U.h.n (/home/nowuser/src/node_modules/firebase/database-node.js:247:178)<br> &nbsp; &nbsp;at Og.h.gf (/home/nowuser/src/node_modules/firebase/database-node.js:184:552)<br> &nbsp; &nbsp;at get (/home/nowuser/src/firebase.js:33:30)<br> &nbsp; &nbsp;at server.get (/home/nowuser/src/index.js:20:5)<br> &nbsp; &nbsp;at Layer.handle (/home/nowuser/src/node_modules/express/lib/router/layer.js:95:5)<br> &nbsp; &nbsp;at Layer.wrapped (/home/nowuser/src/node_modules/newrelic/lib/transaction/tracer/index.js:184:28)<br> &nbsp; &nbsp;at Layer.wrappedLayerHandleRequest [as handle_request] (/home/nowuser/src/node_modules/newrelic/lib/instrumentation/express.js:364:14)<br> &nbsp; &nbsp;at next (/home/nowuser/src/node_modules/express/lib/router/route.js:137:13)<br> &nbsp; &nbsp;at Route.dispatch (/home/nowuser/src/node_modules/express/lib/router/route.js:112:3)</pre>\n</body>\n</html>\n'
```

Our GH repo does have a dot in the name but since [popper.js](https://github.com/FezVrasta/popper.js) is listed as currently using bundlesize I've assumed that this isn't the real cause of the issue here?

**Comment by siddharthkp** [comment]
Ooh that's interesting. 

Let me test it out

**Comment by siddharthkp** [comment]
Hey, I've added a debug mode in `0.9.0`.

Can you try that out and with `bundlesize --debug` and send me the logs 😄

**Comment by siddharthkp** [comment]
Fixed that in `bundlesize@0.9.2`
